src/timer-view/timer-view.html

```diff
@@ -14,7 +14,7 @@
     [[prompt]]
     <div>
       [[secondsRemaining]]
-      <iron-timer id="timer" start-time="1" current-time="{{secondsRemaining}}" on-iron-timer-end="_timerEnd"></iron-timer>
```

Comment:
In case others come across this conversation, it's all a big mistake. I misread the review system and commented on a removed line.